src/Adapter/AdapterInterface.php

```diff
@@ -19,15 +19,20 @@ public function getErrorCount(string $service = 'default'): int;
 
     /**
      * @param string $service
-     *
-     * @return int
+     * @param int $value
+     */
+    public function incrErrorCount(string $service = 'default', int $value = 1): void;
```

Comment:
I prefer to be extremely verbose. Please change the method name to `incrementErrorCount`.